contributing/contributing-code.md

```diff
@@ -149,6 +149,8 @@ It is acceptable for one person to fulfill these requirements, e.g. if an Owner
 
 We now have a bot that will automatically assign Owners to a PR once it is created, and it is likely at least one of these Owners will also be a Reviewer.
 
+Depending on the files/folders involved, Owners include one or more [AMP Working Groups](https://amp.dev/community/governance/#working-groups), such as the groups for [Security & Privacy](https://amp.dev/community/working-groups/security-privacy/) and [User Interface & Accessibility](https://amp.dev/community/working-groups/ui-and-a11y/).
```

Comment:
To make this a bit more aligned with the intention described in the PR description, I might tweak this a bit
```suggestion
Depending on the files/folders involved, Owners include one or more [AMP Working Groups](https://amp.dev/community/governance/#working-groups), which may include the [Security & Privacy](https://amp.dev/community/working-groups/security-privacy/) and [User Interface & Accessibility](https://amp.dev/community/working-groups/ui-and-a11y/) WGs.
```

Reply:
"Owners *include* ... which may *include*" sounds, subjectively, a bit awkward. I think the meaning/intention is already clear in the current wording (the "such as" suggesting it's an example of the groups that may be included). However, feel free to tweak the wording before merging.

Reply:
Maybe something like"Most files and folders are owned by one or more AMP Working Groups, which may include the..."

The way it's written currently does not make it clear why those working groups are being called out. We could even go a step more explicit saying something more along the lines of "Many files and folders are owned by the UI & A11y Working Group, which can review for accessibility.". I don't see much value in mentioning the Security & Privacy WG here at all, as it's not too common for them to be listed as Owners.

Reply:
The confusion may have to do with the term "Owners" -- There are literal folder-based `OWNERS` where reviews are required by a certain working group, as well as code changes of a certain _type_ that would be best made if done in consultation with a working group owning a certain knowledge domain. 

This is why it made sense to me that UI & A11y  and Security & Privacy are called out specifically for their domain areas which may be relevant regardless of code location. In that regard it makes sense to call it out explicitly in documentation instead of via other automated ownership processes because whether a review might be needed for these considerations is not clearly deterministic.

Reply:
Given that, I feel like the Owners section isn't the best place for such a callout to land. This part of the docs is focused around explaining that owners will be assigned in general by the bot. I think elsewhere in the doc might be more appropriate to mention that some changes should get a11y or security/privacy reviews, but I feel like that doesn't tie into the Owners system directly.

Reply:
The current "Code review and Approval" seems like the most appropriate location for this text. The Owners system is described in a different section called "Roles". We can update the text as is to be more clearly distinct:
```
Please note that some Owners may be relevant regardless of code location and therefore may 
include one or more AMP Working Groups such as the groups for Security & Privacy and 
User Interface & Accessibility. 
```

Reply:
+1 to the "Code Review & Approval" section suggestion. That updated text LGTM, thanks @caroqliu

Reply:
Moved the text to the note as suggested by @caroqliu

Reply:
Thank you! @rcebulko can you help with CLA again? And then we will be good to merge.

Reply:
Actually it looks like prettier is complaining too: @TetraLogicalHelpdesk https://travis-ci.org/github/ampproject/amphtml/jobs/729223189